### PR TITLE
Updated build instructions

### PR DESCRIPTION
**Pull request by Christian-Prather**
Updated build docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +66,36 @@ How to get your contributions merged smoothly and quickly.
...
+
+## Releasing new version (FOR LUCI EMPLOYEES ONLY) ##
+When a new version of this package is ready to be released there are a couple steps to follow. It is important to note that most of the process is automated for convinence and the process should be just a couple of button clicks. 
+
+### Steps ### 
+1. Update release version
+    - This should be its own separate PR and should only update the package.xml `<version> </version>` tag. 
+    - LUCI follows [semver](https://semver.org/) style versioning so MAJOR.MINOR.PATCH versions are expected.
+    - It is okay to not put out versions until multiple changes have happened to the code. 
+2. Once the version increment is merged you simply need to create an official release in github. Make sure you make the release version the same as what is now in `package.xml` we have choses to keep github release and package version in sync.
```
Comment (alex-luci) [inline README.md:95]:
```suggestion
2. Once the version increment is merged you simply need to create an official release in github. Make sure you make the release version the same as what is now in `package.xml`. We have chosen to keep github release and package version in sync.
```

File: README.md [hunk 2]
```diff
@@ -66,3 +66,36 @@ How to get your contributions merged smoothly and quickly.
...
+    - This should trigger an action to auto run called `Create and Sign Package` which you can monitor in the github actions panel. This should grab the released code, build it, make an installable .deb file, gdb sign it an push it to jrog artifactory.
```
Comment (alex-luci) [inline README.md:96]:
```suggestion
    - This should trigger an action to auto run called `Create and Sign Package` which you can monitor in the github actions panel. This should grab the released code, build it, make an installable .deb file, gdb sign it and push it to jrog artifactory. 
```

File: README.md [hunk 3]
```diff
@@ -66,3 +66,36 @@ How to get your contributions merged smoothly and quickly.
...
+
+At this point any node should be available to your system through a `ros2 run` command.
+
+<b>NOTE: Theres is also a provided `build-package.sh` script that can be used to build the ROS package and an installable `.deb` file. This is what the LUCI team uses for its automatic deployments.</b>
+
+<b>Please be aware that the `deploy-package.sh` and the `sign-package.sh` scripts are NOT to be ran by anyone outside of the LUCI company. They will fail if ran by anyone without proper credentials and are only for the automated release process used by LUCI. </b>
+
+
+## Releasing new version (FOR LUCI EMPLOYEES ONLY) ##
+When a new version of this package is ready to be released there are a couple steps to follow. It is important to note that most of the process is automated for convinence and the process should be just a couple of button clicks.
```
Comment (jeredhdean) [inline README.md:88]: Spelling: "Convenience"

File: README.md [hunk 4]
```diff
@@ -66,3 +66,36 @@ How to get your contributions merged smoothly and quickly.
...
+
+### Steps ### 
+1. Update release version
+    - This should be its own separate PR and should only update the package.xml `<version> </version>` tag. 
+    - LUCI follows [semver](https://semver.org/) style versioning so MAJOR.MINOR.PATCH versions are expected.
+    - It is okay to not put out versions until multiple changes have happened to the code. 
+2. Once the version increment is merged you simply need to create an official release in github. Make sure you make the release version the same as what is now in `package.xml` we have choses to keep github release and package version in sync.
+    - This should trigger an action to auto run called `Create and Sign Package` which you can monitor in the github actions panel. This should grab the released code, build it, make an installable .deb file, gdb sign it an push it to jrog artifactory. 
+
+If everything went smoothy congratulation the new package will be released and publicly distributable.
```
Comment (jeredhdean) [inline README.md:98]: Spelling: "If everything went smoothly congratulations the new package will be released and publicly distributable."

**Review by jeredhdean — COMMENTED** [review]
Minor spelling fixes and this looks good.

**Review by alex-luci — APPROVED** [review]

**Review by jeredhdean — APPROVED** [review]

**Review by jeredhdean — APPROVED** [review]

**Pull request merged**